app/services/code_climate/projects_summary_service.rb

```diff
@@ -38,12 +39,21 @@ def open_issues_count_average
     end
 
     def ratings
-      code_climate_ratings = code_climate_metrics.pluck(:code_climate_rate).compact
-      code_climate_ratings.each_with_object(Hash.new(0)) do |code_climate_rate, ratings|
-        ratings[code_climate_rate] += 1
+      @code_climate_ratings = code_climate_metrics.pluck(:id,
```

Comment:
I'm wondering if it might be better to use `select` instead of `pluck` so that we can work with hashes instead of arrays that make it hard to understand what is in `result[0]` for example. On the other hand I think `pluck` is more performant.. but in this case clarity might be more important than performance 🤔 thoughts?

Reply:
whenever using arrays i tried to use a nomeclature like `id_rate_date` so that `id` is [0], `rate` [1] and `date` [2]